packages/patternfly-3/patternfly-react/less/progressivedisclosure.less

```diff
@@ -0,0 +1,3 @@
+.progressive-disclosure-item-children {
```

Comment:
should have caught this before, sorry. Could you rename to progressive-disclosure-item-pf-children ?